tests/types/isBoolean.test.ts

```diff
@@ -0,0 +1,46 @@
+import isBoolean from "../../src/lib/types/isBoolean";
+
+describe("isBoolean", () => {
+    it("should return true for true", () => {
+        expect(isBoolean(true)).toBe(true);
+    });
+    it("should return true for false", () => {
+        expect(isBoolean(false)).toBe(true);
+    });
+    it("should return true for false", () => {
+        expect(isBoolean(false)).toBe(true);
+    });
+    it("should return true for new Boolean(true)", () => {
+        expect(isBoolean(new Boolean(true))).toBe(true);
+    });
+    it("should return true for new Boolean(false)", () => {
+        expect(isBoolean(new Boolean(false))).toBe(true);
+    });
+    it("should return true for Boolean(true)", () => {
+        expect(isBoolean(Boolean(true))).toBe(true);
+    });
+    it("should return true for Boolean(false)", () => {
+        expect(isBoolean(Boolean(false))).toBe(true);
+    });
+    it("should return true for Boolean()", () => {
+        expect(isBoolean(new Boolean())).toBe(true);
+    });
+    it("should return true for Boolean('true')", () => {
+        expect(isBoolean(new Boolean('true'))).toBe(true);
+    });
+    it("should return false for 'true'", () => {
+        expect(isBoolean('true')).toBe(false);
+    });
+    it("should return false for 'false'", () => {
+        expect(isBoolean('false')).toBe(false);
+    });
+    it("should return false for [new Boolean(true)]", () => {
+        expect(isBoolean([new Boolean(true)])).toBe(false);
+    });
+    it("should return false for [new Boolean(true)]", () => {
+        expect(isBoolean([new Boolean(false)])).toBe(false);
+    });
+    it("should return false for ''", () => {
+        expect(isBoolean("")).toBe(false);
+    });
```

Comment:
I know it's overkill, but test numbers and new Error() since they are the most occuring values, yes Error happens when something is thrown.

Other than that very noice💕